src/estimate.coffee

```diff
@@ -105,6 +113,69 @@ updatePivotalTicket = ({ robot, res, projectId, ticketId, points, room }) ->
 
         if room then robot.messageRoom(room, msg) else res.send msg
 
+slackRequest = (robot, apiMethod) ->
+  url = "#{SLACK_BASE_URL}/#{apiMethod}"
+  robot.http(url)
+    .header("Content-Type", "application/json")
+
+blocks = () ->
+  JSON.stringify(
+    [
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": "This is a mrkdwn section block :ghost: *this is bold*, and ~this is crossed out~, and <https://google.com|this is a link>"
+        }
+      },
+      {
+        "type": "section",
+        "block_id": "options_dropdown",
```

Comment:
See https://api.slack.com/messaging/interactivity/enabling at the bottom for an explanation of `block_id`.  You can give each block an id that can be listened to with `slackInteractions.action { blockId: "options_dropdown" } ...` below.